src/components/ArtistDetails.js

```diff
@@ -0,0 +1,67 @@
+// import { useParams, Link } from 'react-router-dom';
+// import axios from 'axios';
+// import { useState } from "react";
+
+
+// const ArtistDetails = () => {
+//     const [ albumList, setAlbumList ] = useState([]);
+
+//     const {artist_name, artist_id} = useParams();
+
+
+
+//     // const apiKey = "f2173fa361a5723207fc6a1fa4c6af56";
+//     // const apiUrl = "https://quiet-ridge-74718.herokuapp.com/http://api.musixmatch.com/ws/1.1/artist.albums.get";
+//     const url = new URL(apiUrl);
+//         url.search = new URLSearchParams({
+//             apikey: apiKey,
+//             artist_id: 24410788,
+//             g_album_name: 1,
+//             s_release_date: "desc",
+//             page_size: 10,
+//             page: 1
+//         });
+//         fetch(url)
+//             .then(function (response) {
+//                 if (response.ok) {
+//                     return response.json();
+//                 } else {
+//                     throw new Error();
+//                 }
+//             })
+//                 .then(function (jsonData) {
+//                     console.log(jsonData)
```

Comment:
Not a big deal, but you may want to remove your console.logs before submitting,  up to you though. I like knowing when I've console logged things as I go along, but I think for the submission they prefer them being removed!